qa/testing.go

```diff
@@ -249,6 +249,49 @@ func (f ResourceFixture) setupClient(t *testing.T) (*common.DatabricksClient, se
 	}, nil
 }
 
+// Quoting from: https://github.com/hashicorp/terraform-plugin-sdk/blob/main/internal/configs/hcl2shim/values.go#L19
+// "UnknownVariableValue is a sentinel value that can be used
+// to denote that the value of a variable is unknown at this time.
+// RawConfig uses this information to build up data about
+// unknown keys.""
+const UnknownVariableValue = "74D93920-ED26-11E3-AC10-0800200C9A66"
+
+// Reference: HCL2ValueFromConfigValue from https://github.com/hashicorp/terraform-plugin-sdk/blob/main/internal/configs/hcl2shim/values.go#L199
+func GetRawConfig(v interface{}) cty.Value {
```

Comment:
This isn't a good name choice, as this method has nothing to do with configs, only `cty.Value`. How about something like `ConvertToValue`?